src/cloco-client.ts

```diff
@@ -150,12 +151,27 @@ export class ClocoClient {
     private async initializeApplication(): Promise<void> {
 
       Logger.log.debug(`ClocoClient.initializeApplication: start.`);
+      let filename: string = `~/.cloco/cache/application_${this.options.application}`;
 
       try {
         this.app = await ApiClient.getApplication(this.options);
+
+        // write to disk cache if in options.
+        if (this.options.useDiskCaching) {
+          Logger.log.debug(`ClocoClient.initializeApplication: writing application ${this.options.application} to disk.`);
+          let s: any = shell as any; // incorrect typing.
+          s.echo(JSON.stringify(this.app)).to(filename);
```

Comment:
why you dropping to shell here? node has a couple of ways to write to file without busting out a separate process. . . .

starting with the built-in https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback

```
fs.writeFile('message.txt', 'Hello Node.js', (err) => {
  if (err) throw err;
  console.log('It\'s saved!');
});
```